python/lsst/ts/Dome/llc_configuration_limits/amcs_configuration_limits.py

```diff
@@ -0,0 +1,85 @@
+from .abstract_llc_configuration_limits import AbstractLlcConfigurationLimits
+import math
+
+DEGREES_TO_RADIANS = math.pi / 180.0
+
+
+class AmcsConfigurationLimits(AbstractLlcConfigurationLimits):
+    """This class holds the limits of the configuration values for the AMCS lower level component. It will
+    validate any set of configuration parameters againt these limits. It will also convert any
+    configuration parameter values expressed in deg/s^n to rad/s^n (with n = 1, 2 or 3).
+    """
+
+    def __init__(self):
+        self.jmax = 3.0  # Maximum jerk in deg/s^3
+        self.amax = 0.75  # Maximum acceleration in deg/s^2
+        self.vmax = 1.5  # Maximum velocity in deg/s
+
+    def validate_and_convert_from_degrees_to_radians(self, configuration_parameters):
+        """Validates the data are against the configuration limits of the lower level component. If
+        necessary it also converts the values expressed in deg/s^n to rad/s^n (with n = 1, 2 or 3).
+
+        Parameters
+        ----------
+        configuration_parameters: `dict`
+            The configuration parameters to validate and possibly convert.
+
+        Returns
+        -------
+        converted_configuration_parameters: `dict`
+            The converted configuration parameters.
+        """
+        # This dict will hold the converted values which we will return at the end of thius function if all
+        # validations are passed.
+        converted_configuration_parameters = {}
+
+        # Validate the provided value for max jerk (in deg/s^3) against the limit (in deg/s^3).
+        if configuration_parameters["jmax"] <= self.jmax:
+            # If the value is smaller than or equal to the limit, convert to rad/s^3.
+            converted_configuration_parameters["jmax"] = (
+                configuration_parameters["jmax"] * DEGREES_TO_RADIANS
+            )
+            # Remove the key so we can check later if any unknown configuration parameters remain.
+            del configuration_parameters["jmax"]
```

Comment:
That's a good suggestion and will work for now. However, the list of defined parameters may get large and I may switch to a dict then to avoid duplicating the list of expected keys.